crates/turborepo/src/main.rs

```diff
@@ -1,32 +1,27 @@
-mod ffi {
-    include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
-}
-
-use std::{ffi::CString, process};
+use std::{env::current_exe, process, process::Stdio};
 
 use anyhow::Result;
 use turborepo_lib::{Args, Payload};
 
-use crate::ffi::{nativeRunWithArgs, GoString};
+fn native_run(args: Args) -> Result<i32> {
+    let mut go_binary_path = current_exe()?;
+    go_binary_path.pop();
+    go_binary_path.pop();
+    go_binary_path.pop();
+    go_binary_path.push("cli");
+    go_binary_path.push("turbo");
 
-impl TryInto<GoString> for Args {
-    type Error = anyhow::Error;
+    let serialized_args = serde_json::to_string(&args)?;
 
-    fn try_into(self) -> std::result::Result<GoString, Self::Error> {
-        let json = serde_json::to_string(&self)?;
-        let cstring = CString::new(json)?;
-        let n = cstring.as_bytes().len() as isize;
+    let mut command = process::Command::new(go_binary_path)
+        .arg(serialized_args)
+        .stdout(Stdio::inherit())
+        .stderr(Stdio::inherit())
+        .spawn()
+        .expect("Failed to execute turbo.");
```

Comment:
I think `build.rs` should copy the Go binary to `OUT_DIR` and have our Rust script expect the Go binary to reside in the same directory.

Reply:
OUT_DIR appears to be a different dir than `target/debug/`. Is there a way to have the contents copied back to `target/debug/` so that the go binary stays next to the rust one?